makefile

```diff
@@ -237,9 +240,7 @@ bookstore: classes.sql all-books
 	for f in `sqlite3 $^ "select '$(BOOK_DIR)/' || semester || '-' || department || '-' || code || '-' || section || '.csv' from section"`; do $(MAKE) $$f; done;
 
 $(EXAM_DIR) $(GRADE_DIR) $(BOOK_DIR) $(SECTION_DIR) $(CATALOG_DIR) images webpages:
-	mkdir $@
-
-
+	mkdir -p $@
```

Comment:
`$@` should be quoted unless you can guarantee that `$(EXAM_DIR)` will always pack it's own quotes

Reply:
due to how make works, the rules for `$(EXAM_DIR)` and company will fail no matter what if they have spaces; see https://github.com/jyn514/GradeForge/blob/master/makefile#L3. this might need better documentation.